packages/connection-manager/index.js

```diff
@@ -0,0 +1,5 @@
+import {
+  default as ConnectionManagerClass
+} from "./src/js/ConnectionManager/ConnectionManager";
+
+export const ConnectionManager = new ConnectionManagerClass();
```

Comment:
I think the package should export a lot more. Otherwise how'd I create a connection? Had to export additional stuff to make it working with the `mesos-client` package I'm working on.
```
import {
  default as ConnectionManagerClass
} from "./src/js/ConnectionManager/ConnectionManager";
import XHRConnection from "./src/js/Connection/XHRConnection";
import * as ConnectionConstants from "./src/js/Connection/ConnectionConstants";

const ConnectionManager = new ConnectionManagerClass();

export { ConnectionConstants, ConnectionManager, XHRConnection };
```